doc/modules/manifold.rst

```diff
@@ -513,12 +514,16 @@ divergence could increase during this phase. Usually it does not have to be
 tuned. A critical parameter is the learning rate. If it is too low gradient
 descent will get stuck in a bad local minimum. If it is too high the KL
 divergence will increase during optimization. More tips can be found in
-Laurens van der Maaten's FAQ (see references).
+Laurens van der Maaten's FAQ (see references). The last parameter, angle,
+is a tradeoff between performance and accuracy. Larger angles imply that we
+can approximate larger regions by a single point,leading to better speed
+but less accurate results. 
 
-Standard t-SNE that has been implemented here is usually much slower than
+The Barnes-Hut t-SNE that has been implemented here is usually much slower than
```

Comment:
Yeah, at least for the toy datasets: the speed up is only ~2-3x on the 2000 digits examples, which still makes it the slowest. Perhaps it's competitive with the next-slowest, MDS?